src/app/auth/components/authentication/authentication.component.ts

```diff
@@ -633,14 +633,19 @@ getCookie(key: string){
     this.showSpinner = true;
     this.loggedrs = false;
     this.scale = true;
+
     if (this.authForm.valid && this.cookie.get('satt_cookies') === 'pass') {
-      // const noredirect = 'true';
       this.authService
         .login(this.f.email?.value, this.f.password?.value)
         .pipe(
           takeUntil(this.onDestroy$),
           catchError((error: HttpErrorResponse) => {
-            if (error.error.error.message === 'user not found') {
+            if (
+              error.error.message ===
+              'ValidationError: "password" failed custom validation because password not match'
```

Comment:
n this error checking condition, it might be more robust to check if the error message starts with 'ValidationError' rather than matching the entire error string. This way, our condition isn't tightly coupled to a specific error message and we can catch more generic validation errors. 

you can use this check error.error.message.startsWith('ValidationError')